frontend/app/src/App.vue

```diff
@@ -347,7 +349,24 @@ export default defineComponent({
       }
     });
 
+    const { language } = setupSettings();
+
+    onBeforeMount(() => {
+      if (get(language) !== root.$i18n.locale) {
```

Comment:
I guess it will be some effort to migrate to the bridge (last time I checked the bridge didn't support vue 2.7) or vue-18n-composable in preparation for vue3 and vue-i18n v9. 

Do you think it can be done as part of this PR?

Reply:
Now that I think of it better not make it part of this PR, we need to coordinate and make the changes in a new PR otherwise it's gonna be hard to sync stuff as we make changes into components.